tools/run_tests/python_utils/jobset.py

```diff
@@ -473,7 +474,10 @@ def finish(self):
     while self._running:
       if self.cancelled(): pass  # poll cancellation
       self.reap()
-    if platform_string() != 'windows':
+    # Clear the alarms when finished to avoid a race condition causing job
+    # failures. Don't do this when running multi-VM tests because clearing
```

Comment:
Out of curiosity, why exactly clearing the alarms for multi-VM tests causes tests to stall?